radis/lbl/broadening.py

```diff
@@ -1339,25 +1341,10 @@ def _calc_lineshape(self, dg):
             )
 
         if __debug__:
-            t2 = time()
-            if self.verbose >= 3:
-                printg("... Initialized vectors in {0:.1f}s".format(t1 - t0))
-                if broadening_method == "voigt":
-                    printg(
-                        "... Calculated Voigt profile (jit={1}) in {0:.1f}s".format(
-                            t2 - t1, jit
-                        )
-                    )
-                elif broadening_method == "convolve":
-                    printg(
-                        "... Calculated Lorentzian profile in {0:.1f}s".format(t11 - t1)
-                    )
-                    printg(
-                        "... Calculated Gaussian profile in {0:.1f}s".format(t12 - t11)
-                    )
-                    printg("... Convolved both profiles in {0:.1f}s".format(t2 - t12))
-                elif broadening_method == "fft":
-                    raise NotImplementedError("FFT")
+            _time["t2"] = time()
+            self.time._calc_lineshape_time(
```

Comment:
Time()  should be a very general object and shouldn't have methods specific for, for instance the lineshapes. 
It could be something like : 


```
if broadening_method == "voigt":                 
   Self.time.stop(level=3, "... Calculated Voigt profile (jit={1})") 
```

Which would handle the verbose, print, and store?